scripts/projector.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# failfast
+set -e
+
+# Define these 2 optional variables if you want to customize the default installation,
+# then copy-paste the remainder of the script:
+# - the name of the projector config (defaults to 'Idea')
+# projector_config_name=Idea
+# - the name of the IDE to install (defaults to "IntelliJ IDEA Community Edition 2020.3.4")
+# Note: to get list of available IDEs, launch `projector find`
+# projector_ide_name="IntelliJ IDEA Community Edition 2020.3.4"
+
+projector_config_name=${projector_config_name:-Idea}
+projector_ide_name=${projector_ide_name:-IntelliJ IDEA Community Edition 2020.3.4}  # Latest projector-tested IDE
+
+# Install projector-installer
+# https://github.com/JetBrains/projector-installer
+apt-get update
+apt-get install -y --no-install-recommends python3 python3-pip python3-cryptography less libxext6 libxrender1 libxtst6 libfreetype6 libxi6
+pip3 install projector-installer
+
+# Setup projector
+sudo -u ubuntu projector --accept-license ide autoinstall --config-name "${projector_config_name}" --ide-name "${projector_ide_name}" --hostname {{ .STRIGO_RESOURCE_DNS }}
+# Cannot use password as it needs to be in the URL as query parameter 'token'
+# and it cannot be different for each session in Strigo web interfaces
+# cat << EOF >> "~ubuntu/.projector/configs/${projector_config_name}/config.ini"
+# [PASSWORDS]
+# password = {{ .STRIGO_WORKSPACE_ID }}
+# ro_password = {{ .STRIGO_WORKSPACE_ID }}
+# EOF
+# sudo -u ubuntu projector --accept-license config rebuild "${projector_config_name}"
+
+# Create, enable and start projector config service
+cat << EOF > /lib/systemd/system/projector@.service
+[Unit]
+Description=Jetbrains Projector - ${projector_config_name}
+After=network.target
+
+[Service]
+Type=exec
+ExecStart=/home/ubuntu/.projector/configs/${projector_config_name}/run.sh
+Restart=always
+User=%i
+
+[Install]
+WantedBy=default.target
+EOF
+systemctl enable --now projector@ubuntu
+
+# Display projector config password in terminal
+# cat <<\EOF > /etc/profile.d/projector-terminal.sh
+# if [ $USER = ubuntu ]; then
+#   echo -ne '\nProjector '
+#   grep '^password = ' "~/.projector/configs/${projector_config_name}/config.ini"
+#   echo
+# fi
+# EOF
+
+# Add user settings
+sudo -u ubuntu mkdir --parent ~ubuntu/.local/share/JetBrains/consentOptions/
+sudo -u ubuntu touch ~ubuntu/.local/share/JetBrains/consentOptions/accepted
+echo -n "rsch.send.usage.stat:1.1:0:$(date +%s%3N)" > ~ubuntu/.local/share/JetBrains/consentOptions/accepted
```

Comment:
what kind of consent is automatically set here? Is this some sort of telemetry that one should be able to deactivate?

suggestion: add an empty line at the end of the file so that this script piece can be copy-pasted in a strigo initialization script without side-effects (2 script lines in one line)

Reply:
It's to avoid this "data sharing" prompt screen when opening projector:
![image](https://user-images.githubusercontent.com/175128/132131429-97664f44-b127-4044-abdb-6d1d0ae4788f.png)

Don't think this file is supposed to contain any other line.
Anyway, projector do not skip the data sharing prompt screen as soon as this file contains a new line, so not possible to add one.

Reply:
ok, but the current code seems to allow sending telemetry by default. Is it possible to deactivate the data sharing prompt by disabling telemetry?

The other suggestion in my previous message had nothing to do with this topic (sorry for the confusion). It was just to add an empty line at the end of `scripts/projector.sh`.

Reply:
It is disabling the telemetry (this is the meaning of the last `:0:`)

Reply:
(looking at the code comment in your last commit, it looks like this setting already disables telemetry - it is just not obvious from this line of code)